packages/server/__snapshots__/3_plugins_spec.coffee.js

```diff
@@ -29,6 +27,8 @@ Error: Async error from plugins file
     at stack trace line
     at stack trace line
     at stack trace line
+
+Error: Async error from plugins file
```

Comment:
I will need to see why this snapshot needs to be updated. As far as i understand, by updating the desktop-gui in `open` mode it should **not** affect how errors are displayed in `run` mode. If that were the case then lots of snapshots would need to be updated. I'm confused as to why there's only a single update necessary here, and it's possible there's been a regression.

Reply:
This update comes from [the addition of `err.details`](https://github.com/cypress-io/cypress/blob/issue-2955-update-desktop-gui-errors/packages/server/lib/modes/run.coffee#L358). Previously, those details would have been logged as part of `err.stack` but when I removed them  from `err.message` the details were no longer logged in `err.stack`. I'll do some looking to make sure that's the right fix though.